### PR TITLE
Implement new commands for Crab Arm flap out and Spatula flip.

### PR DESCRIPTION
**Pull request by pak-plau**

### CODE REVIEW

**Comment by PrathamRawat** [comment]
Rejected

File: src/org/usfirst/frc/team694/robot/RobotMap.java [hunk 1]
```diff
@@ -21,11 +21,19 @@
     public static final int DRIVETRAIN_LEFT_TOP_MOTOR_PORT = 3;
     public static final int DRIVETRAIN_LEFT_MIDDLE_MOTOR_PORT = 2;
     public static final int DRIVETRAIN_LEFT_BOTTOM_MOTOR_PORT = 1;
+    
+    /**************************************************************************
+     * Spatula and CrabArm Constants         //All times are in seconds
+     *************************************************************************/
+    public static final double PRE_FLIP_WAIT_TIME = 2;
+    public static final double POST_FLIP_WAIT_TIME = 2;
+    public static final double FLAP_OUT_SPEED = -0.5;
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/RobotMap.java]: Should be positive. Negate it in the Crab arm

File: src/org/usfirst/frc/team694/robot/commands/CrabArrowAcquireCommand.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
     //CrabArrow = CrabArms and Spatula
 
     public CrabArrowAcquireCommand() {
+        addSequential(new SpatulaFlipDownIfUpCommand());
         addParallel(new CrabArmAcquireCommand());
         addSequential(new SpatulaAcquireCommand());
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/commands/CrabArrowAcquireCommand.java:11]: Should this really be sequential?

File: src/org/usfirst/frc/team694/robot/subsystems/CrabArm.java [hunk 3]
```diff
@@ -30,7 +30,7 @@ public void acquire() {
     }
 
     public void flapOut() {
-        crabArmMotors.set(-1);
+        crabArmMotors.set(RobotMap.FLAP_OUT_SPEED);
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/subsystems/CrabArm.java]: As stated before, keep the constant constant and negate this (-1 * FLAP_OUT_SPEED)

**Review by TacoTechnica — CHANGES_REQUESTED** [review]
Almost there, only minor changes

**Review by TacoTechnica — APPROVED** [review]
👍

**Pull request merged**